pennylane/transforms/batch_input.py

```diff
@@ -58,11 +57,11 @@ def batch_input(
 
         dev = qml.device("default.qubit", wires = 2, shots=None)
 
-        @batch_input(argnum=0)
+        @batch_input(argnum=1)
```

Comment:
I already thought this before, very minor though: Should we do
```suggestion
        @qml.batch_input(argnum=1)
```
instead, to allow for easy copy-pasting of this code snippet, without having to import `batch_input` separately to `qml`,which anyways is needed? :)